central/plugins/org.jboss.tools.central/src/org/jboss/tools/central/internal/browser/VersionedBrowser.java

```diff
@@ -50,6 +55,12 @@ public VersionedBrowser(Composite parent, int style) {
 		String result = (String) evaluate(browserScript);
 		name = result.substring(0, result.indexOf("_")); //$NON-NLS-1$
 		version = result.substring(result.indexOf("_") + 1); //$NON-NLS-1$
+		
+		if (!usageReported) {
```

Comment:
If the fix is trivial, yes, why not. But I don't see it as a blocker issue so it can wait if non-trivial